app/presenters/queries/expand_dependents.rb

```diff
@@ -21,7 +21,9 @@ def dependents
         links.group_by(&:first).each_with_object({}) do |(type, link_array), hash|
           reverse = ::Queries::DependeeExpansionRules.reverse_name_for(type).to_sym
           link_ids = link_array.map(&:last)
-          items = all_web_content_items.select { |item| link_ids.include?(item.content_id) }
+          items = all_web_content_items.select do |item|
+            link_ids.include?(item.content_id)
+          end
```

Comment:
This looks to have nothing to do with what the commit is about? It also looks to be just a stylistic change (if I haven't missed anything)?

If it is just stylistic, I find that committing these separately can help make reviewing easier, as its more informative for those doing reviewing.